package.json

```diff
@@ -23,14 +23,10 @@
     "prop-types": "^15.7.2"
   },
   "scripts": {
-    "build": "webpack --mode=production",
-    "dev": "concurrently \"npm run server\" \"npm run frontend\"",
-    "develop": "webpack --mode=development",
+    "server": "concurrently \"npm run server\" \"npm run frontend\"",
```

Comment:
The new `server` should run `concurrently "npm run backend" "npm run frontend"` because we changed the name around

Reply:
If you run `npm run server` at this state, you will get infinite loop because `npm run server` runs `npm run server` 😎